packages/http/fetch/src/middlewares/browser/middlewareFactory.ts

```diff
@@ -29,12 +29,7 @@ export class MiddlewareFactory {
 	public static getDefaultMiddlewareChain(customFetch: (request: string, init: RequestInit) => Promise<Response> = fetch as any): Middleware[] {
 		// Browsers handles redirection automatically and do not require the redirectionHandler
 
-		const middlewareArray: Middleware[] = [];
-		const retryHandler = new RetryHandler(new RetryHandlerOptions());
-		middlewareArray.push(retryHandler);
-
-		middlewareArray.push(new CustomFetchHandler(customFetch));
-
+		const middlewareArray: Middleware[] = [new RetryHandler(), new ParametersNameDecodingHandler(), new CustomFetchHandler(customFetch)];
```

Comment:
The problem with using a handler is that it can be easily  missed in the custom middleware chains.

Reply:
Are you saying we should not use a handler to decode the parameters names?